index.html

```diff
@@ -124,71 +124,9 @@ <h2>Concepts</h2>
     <h3><dfn>Processing Units</dfn></h3>
     <p>
       Computing devices consist of a multitude of different processing units such as the Central
-      Processing Unit (<dfn>CPU</dfn>), Graphics Processing Unit (GPU) and many specialized
+      Processing Unit (CPU), Graphics Processing Unit (GPU) and many specialized
```

Comment:
This sentence doesn't read well. Is there a missing dot?

Reply:
Added a "the"

Reply:
I'm still confused. How about this?

```
      Computing devices consist of a multitude of different processing units such as the Central
      Processing Unit (CPU), the Graphics Processing Unit (GPU) and many specialized
      processing units. The latter are becoming popular such as ones designed to accelerate specific tasks like
```